server/manifest.go

```diff
@@ -111,10 +116,9 @@ func WriteManifest(name model.Name, config *Layer, layers []*Layer) error {
 	defer f.Close()
 
 	m := Manifest{
-		SchemaVersion: 2,
-		MediaType:     "application/vnd.docker.distribution.manifest.v2+json",
-		Config:        config,
-		Layers:        layers,
+		MediaType: "application/vnd.docker.distribution.manifest.v2+json",
```

Comment:
While counter-intuitive we may want to continue to add these for now otherwise `ollama push` may fail (worth trying)

Reply:
Yes - you are correct. I removed the patch related to SchemaVersion so we will continue to write it but not verify. I had been looking for the history of this field given that it was already version 2 from the beginning.